exo-go/src/aws_helpers/terraform_helpers.go

```diff
@@ -0,0 +1,82 @@
+package awsHelper
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+// InitAccount prepares a blank AWS account to be used with Terraform
+func InitAccount(bucketName, tableName, region string) error {
+	config := aws.NewConfig().WithRegion(region)
+	session := session.Must(session.NewSession())
+
+	err := createRemoteState(session, config, bucketName)
+	if err != nil {
+		return err
+	}
+
+	return createLockTable(session, config, tableName)
+}
+
+// creates s3 bucket to store terraform remote state
+func createRemoteState(currSession *session.Session, config *aws.Config, bucketName string) error {
+	s3client := s3.New(currSession, config)
+
+	hasBucket, err := hasBucket(s3client, bucketName)
+	if err != nil {
+		return err
+	}
+
+	if !hasBucket {
```

Comment:
Thoughts on changing this to 
```
if hasBucket {
  return nil 
}
```
Then the rest of the function can be unindented.

Same for hasTable in the createLockTable function